### ISSUE TITLE
Update the archetype

### ISSUE THREAD

**Issue by willwade** [openassistive/OpenATFrontEnd#22]
i.e https://github.com/openassistive/OpenATFrontEnd/blob/master/archetypes/item.md
My gut feeling is this behind what our backend API is validating.. Am I right @hosseinamin ?

**Comment by hosseinzoda** [comment]
@willwade Yes. I think, That file should have placeholder for all required or optional input.

**Comment by willwade** [comment]
Made a start on this in https://github.com/openassistive/OpenATFrontEnd/commit/436f277c44b3847fe6164dfb13b4da70e95aea7e

**Comment by willwade** [comment]
I'm also working on a commented version - https://www.dropbox.com/s/57yrpxpf9oip35p/3DHeadpointer-y8f.md?dl=0 - note this isn't working as hugo new item/somefile - not sure why
